facebookincubator/create-react-app
# Create git repository with initial commit

Fixes #1244

The intended test plan was:

1. Create a new React application on a machine with no Git installed. The application should be created with no additional artifacts just like before.
1. Create a new React application in a folder that is not part of a git repository. The application should be created with an additional new git repository. The initial application source files should be committed to the git repository as the initial commit.
1. Create a new React application in a folder that is already part of an existing git repository. The application should be created with no additional artifacts just like before.

However because I am running Windows natively I was unable to test this the way I wanted because the ```cra.sh``` doesn't run. Trying to do so in a Docker based Linux container was also less than successful.
